logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/conversation/folder/RemoveConversationFromFavoritesUseCase.kt

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
change: name of the file is `RemoveConversationFromFaroritesUseCase` without `Test` at the end 😄